src/sandstorm/run-bundle.c++

```diff
@@ -1296,6 +1296,8 @@ private:
       unshareUidNamespaceOnce();
     }
 
+    KJ_SYSCALL(unshare(CLONE_NEWCGROUP));
```

Comment:
What happens to people running older kernels that don't understand `CLONE_NEWCGROUP`? Granted, this flag is quite old now, but I'd hate for someone who has had a sandstorm server running on an ancient kernel just fine up until now find it suddenly breaks when they auto-update.

Reply:
I suppose we could treat this as non fatal and just disable use of cgroups if it does fail.

Reply:
Pushed a commit that changes this; if unshare returns EINVAL it just skips cgroup setup, which should do the right thing on older kernels.